linux_os/guide/services/ssh/ssh_server/sshd_disable_root_login/rule.yml

```diff
@@ -56,6 +56,16 @@ references:
 
 {{{ complete_ocil_entry_sshd_option(default="no", option="PermitRootLogin", value="no") }}}
 
+{{% if product == "rhel8" %}}
+platforms:
+    - not_ovirt-host
+
+warnings:
+    - general: |-
+        This rule is disabled on Red Hat Virtualization Hosts, it will report not applicable.
+        RHV hosts require root access to be managed by RHV Manager.
```

Comment:
@almusil Let me know precise this is.

Any info about other rule is appreciated too.

Reply:
This sounds like a good explanation.